pkg/graveler/settings/manager.go

```diff
@@ -71,20 +95,55 @@ func (m *Manager) Save(ctx context.Context, repositoryID graveler.RepositoryID,
 	if err != nil {
 		return err
 	}
-	err = m.blockAdapter.Put(ctx, block.ObjectPointer{
+
+	data := Settings{
+		Identifier: fmt.Sprintf(settingsUniqueRelativeKey, m.committedBlockStoragePrefix, key, xid.NewWithTime(time.Now())),
```

Comment:
That's the default behaviour
```suggestion
		Identifier: fmt.Sprintf(settingsUniqueRelativeKey, m.committedBlockStoragePrefix, key, xid.New()),
```